SimPEG/EM/FDEM/RxFDEM.py

```diff
@@ -11,17 +12,27 @@ class BaseRx(SimPEG.Survey.BaseRx):
     """
 
     def __init__(self, locs, orientation=None, component=None):
-        assert(orientation in ['x','y','z']), "Orientation {0!s} not known. Orientation must be in 'x', 'y', 'z'. Arbitrary orientations have not yet been implemented.".format(orientation)
-        assert(component in ['real', 'imag']), "'component' must be 'real' or 'imag', not {0!s}".format(component)
+        assert(
+            orientation in ['x', 'y', 'z']
+        ), "Orientation {0!s} not known. Orientation must be in 'x', 'y', 'z'."
+        " Arbitrary orientations have not yet been implemented.".format(
+            orientation
+        )
+        assert(
+            component in ['real', 'imag']
```

Comment:
it could be - I think it is odd to treat them unequally though. Keeping it as a string, I could see us allowing:
`["real", "in-phase"]` and `["imag", "imaginary", "quadrature", "out-of-phase" ]`

@dougoldenburg: any suggestions on what to label `real` and `imag` data (does "component" make sense, or do you have another suggestion?)